### ISSUE TITLE
Timeout for visualizer with custom dataset

### ISSUE THREAD

**Issue by humzaiqbal** [jakebian/quiver#40]
So I'm running the visualizer on a dataset of `(None, 8, 200, 1)` images with 50 datasets. I want to display the predictions for each image but when I pass in the classes `range(1, 51) as follows
`server.launch(model, range(1, 51), top=4, port=5001)`, and I try to look at the output for a particular image and layer, it does not display anything

**Comment by jakebian** [comment]
are there error logs in the terminal?

**Comment by humzaiqbal** [comment]
No, at the bottom below the list of images in my directory there is just this loading icon
<img width="605" alt="screen shot 2017-01-09 at 9 08 12 pm" src="https://cloud.githubusercontent.com/assets/10646338/21794522/d7bc3090-d6af-11e6-8180-2433f9c10bb0.png">

**Comment by jakebian** [comment]
You're saying your terminal window is not logging any  errors at all when this happens?

**Comment by humzaiqbal** [comment]
nope, the terminal is just displaying a GET request for a particular output

**Comment by jakebian** [comment]
Is there any errors in the javascript console in the browser?

**Comment by humzaiqbal** [comment]
Yeah there is. It says `Uncaught (in promise) TypeError: e[1].split is not a function`
